### PR TITLE
otvdm@0.9.0: Fix nested directory installation problem

### LINKED ISSUE #12731: [Bug]: otvdm@0.9.0 is installed under redundant directory named "otvdm-v0.9.0"
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

otvdm

### Expected/Current Behaviour

Scoop installs otvdm app in nested directory named its version number redundantly.
That means normally contents of Scoop apps are installed under ```[app name]\[version number]\```, but otvdm contents are under ```otvdm-v0.9.0``` again.

### Steps to Reproduce

```console
PS> scoop install extras/otvdm
Installing 'otvdm' (0.9.0) [64bit] from extras bucket
Loading otvdm-v0.9.0.zip from cache.
Checking hash of otvdm-v0.9.0.zip ... ok.
Extracting otvdm-v0.9.0.zip ... done.
Linking ~\scoop\apps\otvdm\current => ~\scoop\apps\otvdm\0.9.0
Creating shim for 'otvdm'.
'otvdm' (0.9.0) was installed successfully!
```

### Possible Solution

The distributed compressed file that Scoop gets has nested directory named ```otvdm-v0.9.0``` internally. it is the same name exactly to the redundant directory in the installation directory.
The solution is easy that just **adding extract_dir property** under autoupdate property on its manifest with ```otvdm-v$version```.

In extra, remove directory name from ```bin``` property
Last, the manifest also contains wrong license information. it has to be fixed also.

### Scoop and Buckets Version

```console
PS> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
f9db6a33c (HEAD -> master, origin/master, origin/HEAD) kate: Update to version 23.08.4-2297

'games' bucket:
5827a8ecd (HEAD -> master, origin/master, origin/HEAD) citra: Update to version 2089-20240202

'java' bucket:
4bf5794c (HEAD -> master, origin/master, origin/HEAD) intellij-jbr17: Update to version 17.0.9-b1087.11

'main' bucket:
0669933bb (HEAD -> master, origin/master, origin/HEAD) ttyper: Update to version 1.4.1

'versions' bucket:
7b3ad6289 (HEAD -> master, origin/master, origin/HEAD) vlc-nightly: Update to version 20240202
```

### Scoop Config

```console
PS> scoop config

last_update           : 2024-02-02 18:57:32
debug                 : True
aria2-enabled         : True
aria2-warning-enabled : False
use_external_7zip     : True
scoop_branch          : master
scoop_repo            : https://github.com/ScoopInstaller/Scoop
```

### PowerShell Version

```console
PS> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.4.1
PSEdition                      Core
GitCommitId                    7.4.1
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Additional Softwares

_No response_

### PR DESCRIPTION
**Pull request by Capella87**
Closes #12731 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/otvdm.json [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
+    "license": "GPL-2.0",
     "architecture": {
         "64bit": {
             "url": "https://github.com/otya128/winevdm/releases/download/v0.9.0/otvdm-v0.9.0.zip",
             "hash": "842b11aed5fa81f3e1d4272e0ee7d37f1a5a8f936de825309dda672835e16fd4"
         }
     },
-    "bin": "otvdm-v0.9.0/otvdm.exe",
+    "extract_dir": "otvdm-v0.9.0",
+    "bin": "otvdm.exe",
```
Comment (aliesbelik) [inline bucket/otvdm.json:13]:
```suggestion
    "bin": [
        "otvdm.exe",
        "otvdmw.exe"
    ],
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Wonder if we could add `otvdmw.exe` to bin as well.

**Comment by Capella87** [comment]
I think so.

**Comment by Capella87** [comment]
@aliesbelik I've added otvdmw.exe in ```bin``` property.

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**